doc/experiment.rst

```diff
@@ -329,6 +332,9 @@ Example
   *Generating*
    - the ``Orchestrator`` output directory
    - the ``Model`` output directory
+  *Preview*
```

Comment:
Preview*ing*

Reply:
unless we should take all the ing off the other titles, whatever sounds better to u!

Reply:
changed to previewing